gallery/workbook/microsoft.operationalinsights-workspaces.json

```diff
@@ -135,6 +135,103 @@
 					"name": "Agent Health"
 				}
 			]
+		},
+		{
+			"id": "ServicesHub",
+			"name": "ServicesHub",
+			"templates": [
+				{
+					"id": "Workbooks/ServicesHub/AtResource/ADAssessmentWorkbookTemplate",
+					"name": "Active Directory Assessment Workbook Template",
+					"description": "Recommendations generated from the Active Directory Assessment that ran on your system.",
+					"author": "Microsoft",
+					"isPreview": true
```

Comment:
FYI: you don't need "workbook template' in the name of every item.  that's like naming all your word documents "resume word document.docx"